.github/workflows/wheels.yml

```diff
@@ -45,17 +41,9 @@ jobs:
            git fetch --prune --unshallow
 
       - name: Build wheels for CPython
-        uses: pypa/cibuildwheel@v3.2.1
+        uses: pypa/cibuildwheel@v3.3.0
         with:
           output-dir: dist
-        env:
-          CIBW_BUILD: "cp310-* cp311-* cp312*"
-          CIBW_SKIP: "*-musllinux_*"  #  numpy doesn't have wheels for musllinux so we can't build some quickly and without bloating
-          CIBW_ARCHS_LINUX: "x86_64"
-          MACOSX_DEPLOYMENT_TARGET: "10.9" # as of CIBW 2.9, this is the default value, pin it so it can't be bumped silently
```

Comment:
note that I intentionally didn't replicate this line because the *correct* target depends on the Python version being built against, and it's much easier to rely on cibuildwheel's default for this.